src/Runner.Common/JobServerQueue.cs

```diff
@@ -481,16 +502,13 @@ private async Task ProcessSummaryUploadQueueAsync(bool runOnce = false)
                             telemetryRecord.Issues.Add(issue);
                             QueueTimelineRecordUpdate(_jobTimelineId, telemetryRecord);
 
-                            Trace.Info("Catch exception during summary file upload to results, keep going since the process is best effort.");
+                            Trace.Info("Catch exception during file upload to results, keep going since the process is best effort.");
                             Trace.Error(ex);
-                        }
-                        finally
-                        {
                             errorCount++;
                         }
                     }
 
-                    Trace.Info("Tried to upload {0} summary files to results, success rate: {1}/{0}.", filesToUpload.Count, filesToUpload.Count - errorCount);
+                    Trace.Info("Tried to upload {0} files to results, success rate: {1}/{0}.", filesToUpload.Count, filesToUpload.Count - errorCount);
```

Comment:
```suggestion
                    Trace.Info("Tried to upload {0} file(s) to results, success rate: {1}/{0}.", filesToUpload.Count, filesToUpload.Count - errorCount);
```

Same as one of the earlier Nits